plots/plot_results_in_heatmap.py

```diff
@@ -140,7 +140,8 @@ def __save_plot(figure: Figure, dataset_name: str, chosen_metric: str) -> None:
         os.makedirs(f"{folder_location}")
     
     figure.savefig(
-        f"{folder_location}/heat_map_{time_stamp_string}.png",
+        f"{folder_location}/heat_map.png",
+        # f"{folder_location}/heat_map_{time_stamp_string}.png",
```

Comment:
I noticed that the timestamp has been removed from the filename. Just wanted to understand if there was a specific reason behind this change. I'm also concerned that overwriting files for different combinations and figures could cause an issue.

Reply:
In this way, it is easier to debug the code. Once this part of the code is final we can bring back the time_stamp.

Reply:
OK, thanks for clarifying